source/_integrations/fronius.markdown

```diff
@@ -108,3 +121,7 @@ sensor:
       device: 0
     - sensor_type: power_flow
 ```
+
+## Note
+
+Fronius often provides firmware updates for the datamanager interfaces and the devices in their system, it's recommended to check and apply them regurarly.
```

Comment:
It is in the aspect that some parts of the API used by this integration are missing in older firmware revisions of the datamanager card. I thought it could be useful to point people in the direction of updating it to the lastest before starting to look for issues in HA.
Clarified the statement accordingly.